cms/djangoapps/contentstore/views/course.py

```diff
@@ -283,6 +294,28 @@ def course_rerun_handler(request, course_key_string):
             })
 
 
+@login_required
+@ensure_csrf_cookie
+@require_GET
```

Comment:
I'd like to defer this one as a follow-on story. The current implementations replicates existing patterns in studio, even though there's a desire to migrate to a different framework.